main.go

```diff
@@ -135,13 +137,13 @@ func createFile(eventPullRequestTitle string, eventPullRequestIssuer string, pul
 	return response, err
 }
 
-func jiraTicketsExtraction(pullRequestBody string) string {
-	if len(pullRequestBody) == 0 {
+func jiraTicketsExtraction(jiraTickets string) string {
+	if len(jiraTickets) == 0 {
 		return ""
 	}
 
 	regexPattern := regexp.MustCompile(`(?i)(\b[\\n](?:close|closes):?\b|\b(?:close|closes):?\b).*?\b(LIM-\d+)\b`)
-	matches := regexPattern.FindAllStringSubmatch(pullRequestBody, -1)
+	matches := regexPattern.FindAllStringSubmatch(jiraTickets, -1)
```

Comment:
The rename here doesn't make sense. you receive the body, and return the tickets.

Reply:
Thanks, done